util/compare.go

```diff
@@ -45,6 +45,26 @@ func Compare(a, b interface{}) int {
 		case json.Number:
 			return compareJSONNumber(a, b)
 		}
+	case int:
```

Comment:
Perhaps I didn't pay attention -- what was the reason for this? 🔍

Reply:
I had been hesitant because I've conflated `util.Compare` with `ast.Compare`. This change is OK, I think.